includes/Classifai/Services/LanguageProcessing.php

```diff
@@ -142,6 +142,16 @@ public function register_endpoints() {
 				'permission_callback' => [ $this, 'generate_post_title_permissions_check' ],
 			]
 		);
+
+		register_rest_route(
+			'classifai/v1/openai',
+			'resize-content',
+			[
+				'methods'             => WP_REST_Server::EDITABLE,
+				'callback'            => [ $this, 'resize_content' ],
+				'permission_callback' => [ $this, 'resize_content_permissions_check' ],
```

Comment:
We should add an `args` field here and detail out all the various arguments that can be passed in, similar to what we do on other endpoints